JShellAPI/src/main/java/org/togetherjava/jshellapi/Utils.java

```diff
@@ -0,0 +1,18 @@
+package org.togetherjava.jshellapi;
+
+import org.springframework.util.StringUtils;
+
+public final class Utils {
+    private Utils() {}
+
+    public static boolean validateJShellWrapperImageName(String imageName) {
+        if (!StringUtils.hasText(imageName)
+                || !imageName.endsWith(Config.JSHELL_WRAPPER_IMAGE_NAME_TAG)) {
+            return false;
+        }
+
+        final String imageNameFirstPart = imageName.split(Config.JSHELL_WRAPPER_IMAGE_NAME_TAG)[0];
+
+        return StringUtils.hasText(imageNameFirstPart);
+    }
+}
```

Comment:
Move this to a static private method in Config, since it's only used once and should only be handled by Config.